src/js/components/SharedComponents/CalendarRangeDatePicker.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { PropTypes } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+
+import Moment from 'moment';
+
+function Navbar({
```

Comment:
It looks like there are two components in this file, `NavBar` and `CalendarRangeDatePicker`. Please move `NavBar` to it's own file. (See the components section at https://github.com/fedspendingtransparency/data-act-documentation/blob/master/frontend_apps/SoD.md#overview)